### ISSUE TITLE
The Select component can't be styled

### ISSUE THREAD

**Issue by aditudorache** [Amsterdam/amsterdam-styled-components#1225]
**Describe the bug**
The style of the select component can't be changed

**To Reproduce**
- Try to override the style with `const StyledSelect = styled(Select)...`
- Note that the styling has no effect on the control

**Additional information**
The `className` property is missing from,the top level warpper

**Comment by aditudorache** [comment]
Closing this issue. Can be styled with the `styles.SelectWrapperStyle` selector. It is not obvious though. Some kind of documentation would be nice.

**Comment by jonkoops** [comment]
@aditudorache Yeah, this is a classic example of a component that does way too much. This select element also contains a bunch of unrelated logic such as providing a label, which means that there is no real 'root' element since it's contained in a fragment.

I suggest we keep this issue around for when we refactor this component. I'll add it to the v1 milestone.
